src/pages/Nominate/Active/index.tsx

```diff
@@ -40,8 +38,6 @@ export const Active = () => {
       <PageTitle title={t('nominate.nominate')} />
       <StatBoxList>
         <MinimumActiveBondStatBox />
-        <ActiveNominationsStatBox />
-        <InacctiveNominationsStatBox />
```

Comment:
Why are these not useful on Aleph network?

Reply:
In polkadot you can nominate multiple validators at the same time, so that shows you that you have 7 active and 9 inactive validators. For us you only have 1 validator, and it's status you can see in the box below.